test-project/README.md

```diff
@@ -10,9 +10,9 @@ The test-project module contains **Maven plugin** to run changelog files.
     ### If you are a developer and want to test specific feature on some branch: 
    - At first, you need to choose branch from which you want to create dependency and put to local repository for test purpose. When you chose the branch create jar (dependency) of it <br/>
    To do this, **execute** `mvn clean install` command in the root of the **liquibase-couchbase** module through CLI or using IDE.
-   Make sure that in `pom.xml` of **test-project** the `liquibase-maven-plugin` have the correct version of our created dependency.
-2) Change the properties (URL, username, password for the Couchbase database) in the `src\main\resources\liquibase.liquibase.properties` in the **test-project** module
-3) Write changelog files into the `src\main\resources\liquibase\changelog` directory inside the **test-project** module.
+   Make sure that in `pom.xml` of **test-project** the `liquibase-maven-plugin` has the correct version of our created dependency.
+2) Change the properties (URL, username, password for the Couchbase database) in the `src\main\resources\liquibase.liquibase.properties`.
```

Comment:
I guess src\main\resources\liquibase\liquibase.properties instead of src\main\resources\liquibase.liquibase.properties, right?

Reply:
Yes. Thank you)